sse2neon.h

```diff
@@ -2468,7 +2529,7 @@ FORCE_INLINE void _mm_lfence(void)
 
 // FORCE_INLINE __m128 _mm_shuffle_ps(__m128 a, __m128 b, __constrange(0,255)
 // int imm)
-#ifdef _sse2neon_shuffle
+#if defined(_sse2neon_shuffle) && defined(SSE2NEON_STATEMENT_EXPRESSION)
```

Comment:
I added the `defined(SSE2NEON_STATEMENT_EXPRESSION)` checks so I could force compilers down the other path for testing. These checks should be deleted now, as nothing is defining SSE2NEON_STATEMENT_EXPRESSION. I think we definitely want to keep these separate implementations for compilers that support __builtin_shuffle(vector).

Reply:
ACK, will remove

Reply:
Still present in the latest commit